client/src/Components/Header.js

```diff
@@ -22,7 +22,7 @@ const useStyles = makeStyles(() => ({
   AppBar: {
     height: "5.4em",
     boxShadow: "none",
-    position: "fixed",
+    // position: "fixed",
```

Comment:
May I know the reason why change it to be not fixed? Have you tried it in main page with more than one page of mentions? I guess the header will move when you scroll down?

Reply:
The toggle doesn't show up for some reason if the position is fixed, but I assume we want the header to be stuck on top? Right now the header will not follow if there are multiple mentions. I think this will be a temporary fix for the demonstration to show the toggle, but I can try to have it fixed without blocking any other components moving forward

Reply:
Sry I stated it wrongly. We may NOT want the header to be stuck on top, otherwise it would be like this:
![image](https://user-images.githubusercontent.com/42278237/87100958-4d950380-c213-11ea-8c86-fb9d4e3ab73f.png)
There is an empty strip at the top of the screen, with the left panel displayed it is strange.
The 2nd reason is that the search bar is in the header, and we want the user to have access to the search bar no matter how much she has scroll down. Otherwise she would have to scroll all the way up to the top to enter some new keywords.

Reply:
my bad, yes that's what I meant when I said stuck, that it is always reachable on top by the user. Let me try playing with css before merging.